targets/drive/mobile/main.jsx

```diff
@@ -126,22 +133,91 @@ const isBackgroundServiceParameter = () => {
   return queryDict.backgroundservice
 }
 
-document.addEventListener(
-  'DOMContentLoaded',
-  () => {
-    if (!isBackgroundServiceParameter()) {
-      loadState().then(renderAppWithPersistedState)
+var app = {
+  store: {},
+
+  initialize: function() {
+    this.bindEvents()
+  },
+
+  bindEvents: function() {
+    document.addEventListener(
+      'deviceready',
+      this.onDeviceReady.bind(this),
+      false
+    )
+    document.addEventListener('resume', this.onResume.bind(this), false)
+    document.addEventListener('pause', this.onPause.bind(this), false)
+  },
+
+  startApplication: async function() {
```

Comment:
This function that starts the application is too big and must be refactored. There is too much thing to do.